biooutput.py

```diff
@@ -0,0 +1,101 @@
+import pandas as pd
+import numpy as np
+
+class Biooutput():
+    """Output Class
+
+    """
+
+    def __init__(self, hr_data3):
+        self.data = hr_data3
+
+    def summarize_ecg(self):
```

Comment:
Consider this being a function of the `Bioinput` object (particularly if you position that class as a wrapper of ECG data). It is perfectly reasonable to have an `ECGData.summarize()` function that summarizes parsed internal ECG data of the given instance

Reply:
In this case if the primary action of the function is writing a file to disk, consider including `write_` in the function name to make the intuitive